sale_order_line_date/models/sale_order.py

```diff
@@ -3,28 +3,36 @@
 # © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
 # © 2016 Eficent Business and IT Consulting Services, S.L.
 # Copyright 2017 Serpent Consulting Services Pvt. Ltd.
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 
-from odoo import api, models
+
+from odoo import api, models, fields
 
 
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
     @api.multi
-    @api.onchange('requested_date')
-    def onchange_requested_date(self):
+    # requested_date was renamed to commitment_date
+    @api.onchange('commitment_date')
+    def _onchange_commitment_date(self):
```

Comment:
Is this onchange really needed since the field on `sale.order.line` is now related to the one on `sale.order` ?